docs/api/covidcast-signals/google-survey.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Google Symptom Surveys
+parent: Inactive Signals
+grand_parent: COVIDcast API
+---
+
+# Google Symptom Surveys
+
+* **Source name:** `google-survey`
+* **Number of data revisions since 19 May 2020:** 0
+* **Date of last change:** Never
+
+Data source based on Google-run symptom surveys, through publisher websites,
+their Opinions Reward app, and similar applications. Respondents can opt to skip
+the survey and complete a different one if they prefer not to answer. The survey
+is just one question long, and asks "Do you know someone in your community who
+is sick (fever, along with cough, or shortness of breath, or difficulty
+breathing) right now?" Using this survey data, we estimate the percentage of
+people in a given location, on a given day, that *know somebody who has a
+COVID-like illness*. Note that this is tracking a different quantity than the
+surveys through Facebook, and (unsurprisingly) the estimates here tend to be
+much larger.
```

Comment:
This is slightly outdated, since fb-survey now includes a community signal (that in turn is much larger than the google surveys community signal). Perhaps this:

```suggestion
COVID-like illness*. This estimates a similar quantity to the `*_cmnty_cli` signals from the Symptom Surveys (`fb-survey`) source, but using a different survey population and recruitment method.
```